### PR TITLE
Feat: New service adaptive_learning added

### PR DESCRIPTION
**Pull request by DevendraPPatil**
Feat: New service adaptive_learning added

### CODE REVIEW

File: src/mongo_module/models/adaptiveLearning.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// Define the User schema
+const adaptive_learning = new mongoose.Schema({
+    udise_code : {
+        type: String,
+        required: true,
+        index: true
+    },
+    school_name: {
+        type: String,
+        required: true,
```
Comment (sudeeppr1998) [inline src/mongo_module/models/adaptiveLearning.ts:12]: Set it as false

File: src/mongo_module/models/adaptiveLearning.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        default: Date.now
+    },
+    updatedAt: {
+        type: Date,
+        default: Date.now
+    }
+});
+
+
+// Create the Lesson model
```
Comment (sudeeppr1998) [inline src/mongo_module/models/adaptiveLearning.ts:26]: change comment

File: src/mongo_module/models/adaptiveLearning.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const adaptiveLearning = mongoose.model("adaptive_learning", adaptive_learning);
```
Comment (sudeeppr1998) [inline src/mongo_module/models/adaptiveLearning.ts:27]: change to learnerai_adaptive_learning_enabled

File: src/mongo_module/modules/adaptiveLearning/adaptive_learning.router.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import adaptiveLearningController from "./adaptive_learning.controller";
+
+
+const adaptiveLearningRouter = Router();
+
+adaptiveLearningRouter.post("/addSchoolUdise", adaptiveLearningController.addSchoolUdise);
+
+adaptiveLearningRouter.get("/validateUdise/:udise_code", adaptiveLearningController.validateUdise);
+
+adaptiveLearningRouter.delete("/deleteUdise/:_id", adaptiveLearningController.deleteUdise);
```
Comment (sudeeppr1998) [inline src/mongo_module/modules/adaptiveLearning/adaptive_learning.router.ts:11]: change to delete by udise code

File: src/mongo_module/modules/adaptiveLearning/adaptive_learning.service.ts [hunk 5]
```diff
@@ -0,0 +1,64 @@
+
+import CrudOperations from "../../../common/crud";
+import adaptiveLearning from "../../models/adaptiveLearning";
+
+
+class adaptiveLearningServices {
+
+    // add pointers
```
Comment (sudeeppr1998) [inline src/mongo_module/modules/adaptiveLearning/adaptive_learning.service.ts:8]: update comments

File: src/mongo_module/models/adaptiveLearning.ts (hunk 1 above)
Comment (DevendraPPatil) [reply src/mongo_module/models/adaptiveLearning.ts]: Done

File: src/mongo_module/models/adaptiveLearning.ts (hunk 2 above)
Comment (DevendraPPatil) [reply src/mongo_module/models/adaptiveLearning.ts]: Done

File: src/mongo_module/models/adaptiveLearning.ts (hunk 3 above)
Comment (DevendraPPatil) [reply src/mongo_module/models/adaptiveLearning.ts]: This is not a good practice to give to much bit table name.

File: src/mongo_module/modules/adaptiveLearning/adaptive_learning.router.ts (hunk 4 above)
Comment (DevendraPPatil) [reply src/mongo_module/modules/adaptiveLearning/adaptive_learning.router.ts]: Done

File: src/mongo_module/modules/adaptiveLearning/adaptive_learning.service.ts (hunk 5 above)
Comment (DevendraPPatil) [reply src/mongo_module/modules/adaptiveLearning/adaptive_learning.service.ts]: Done

**Pull request merged**